package.json

```diff
@@ -21,7 +21,8 @@
     "@tryghost/content-api": "1.5.11",
     "nuxt": "2.15.7",
     "nuxt-i18n": "6.28.1",
-    "nuxt-lazy-load": "1.2.6"
+    "nuxt-lazy-load": "1.2.6",
+    "vue-clickaway": "^2.2.2"
```

Comment:
> Published 4 years ago
Ether we should use another liberty or copy/fork the code to our repo. (If you do so, please include a GitHub Link to the original project.)

Reply:
^^ @ClientCrash